DemoNativeControls/app/src/main/java/com/jwplayer/demo/nativecontrols/JWPlayerNativeControls.java

```diff
@@ -0,0 +1,253 @@
+        mPlayerView.removeOnTimeListener(mVideoProgressBar);
+        mPlayerView.removeOnCompleteListener(this);
+    }
+
+    @Override
+    public void onClick(View v) {
+        boolean stoppedPlayback = false;
+        switch (v.getId()) {
+            case R.id.play_and_pause_button:
+                //If we're playing stop us, if we're not playing play us
+                if(mPlayerView.getState() == PlayerState.PLAYING){
+                    mPlayerView.pause();
+                    stoppedPlayback = true;
+                } else if(mPlayerView.getState() != PlayerState.PLAYING){
+                    mPlayerView.play();
+                }
+                break;
+            case R.id.stop_button:
+                //Cancel playback ASAP and reset the UI back to the pre video starts
+                mPlayerView.stop();
+                resetVideoPlaybackUI();
+                break;
+            case R.id.fullscreen_button:
+                if(mPlayerView.getFullscreen()){
+                    mPlayerView.setFullscreen(false,true);
+                } else {
+                    mPlayerView.setFullscreen(true,true);
+                }
+                break;
+            case R.id.fast_forward_button:
+                mPlayerView.seek(mPlayerView.getPosition() + 10);
+                break;
+            case R.id.rewind_button:
+                double newPosition = mPlayerView.getPosition() - 10;
+                //We clamp to 0 because JWPlayer would use negative numbers to jump to <end of video> - seekPos
+                //so -10 is 10 seconds from the end of the video
+                if(newPosition < 0){
+                    newPosition = 0;
+                }
+                mPlayerView.seek(newPosition);
+                break;
+            case R.id.mute_button:
+                if(mPlayerView.getState() == PlayerState.PLAYING){
+                    boolean muted = mPlayerView.getMute();
+                    if(muted){ //we're gonna unmute it
+                        mMuteButton.setText("Mute");
+                    } else { //We're gonna mute it
+                        mMuteButton.setText("UnMute");
+                    }
+                    mPlayerView.setMute(!muted);
+                }
+                break;
+        }
+
+        //Notify all our listeners that one of the mNativePlayerControls has been interacted with
+        for (OnControlsInteraction ocl: mControlChangeListeners) {
+            ocl.OnControlsInteractedWith(stoppedPlayback);
+        }
+    }
+
+    @Override
+    public void onPlay(PlayEvent playEvent) {
+        mPlayPauseButton.setImageResource(R.drawable.exo_controls_pause);
+    }
+
+    @Override
+    public void onPause(PauseEvent pauseEvent) {
+        mPlayPauseButton.setImageResource(R.drawable.exo_controls_play);
+    }
+
+    @Override
+    public void onComplete(CompleteEvent completeEvent) {
+        mPlayPauseButton.setImageResource(R.drawable.exo_controls_play);
```

Comment:
It might be good to use a replay icon here.

Reply:
A replay icon has been swapped in